OpenRA.Mods.Common/HitShapes/Polygon.cs

```diff
@@ -0,0 +1,125 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2017 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Drawing;
+using System.Linq;
+using OpenRA.Graphics;
+using OpenRA.Mods.Common.Graphics;
+
+namespace OpenRA.Mods.Common.HitShapes
+{
+	public class PolygonShape : IHitShape
+	{
+		public WDist OuterRadius { get; private set; }
+
+		[FieldLoader.Require]
+		public readonly int2[] Points;
+
+		[Desc("Defines the top offset relative to the actor's target point.")]
```

Comment:
I know this is just copy-pasted from some other shape, but this is basically bit-rot from times when I wanted to center these around a single custom targetable position instead of CenterPosition.
Replacing 'target point' with 'center' here and below would fix the descriptions. Bonus points if you fix this on the other shapes, too.